src/core/main.c

```diff
@@ -2178,71 +2177,79 @@ int main(int argc, char *argv[]) {
                 /* Disable the umask logic */
                 umask(0);
 
-                /* Make sure that at least initially we do not ever log to journald/syslogd, because it might not be activated
-                 * yet (even though the log socket for it exists). */
+                /* Make sure that at least initially we do not ever log to journald/syslogd, because it might not be
+                 * activated yet (even though the log socket for it exists). */
                 log_set_prohibit_ipc(true);
 
                 /* Always reopen /dev/console when running as PID 1 or one of its pre-execve() children. This is
                  * important so that we never end up logging to any foreign stderr, for example if we have to log in a
                  * child process right before execve()'ing the actual binary, at a point in time where socket
                  * activation stderr/stdout area already set up. */
                 log_set_always_reopen_console(true);
-        }
```

Comment:
This closing brace should stay. The rest should not be under the loop.

Reply:
loop? which loop? not following? this is an if block, and it as partially the same condition as the one following, which is why i merged them

Reply:
Isn't this brace for the loop `for (i = 1; i < argc; i++) {` in line 2127?

Reply:
OK, I see it now, nvm.